### ISSUE TITLE
TypeError: is_enabled() got an unexpected keyword argument 'action'

### ISSUE THREAD

**Issue by binarykitchen** [SublimeCodeIntel/SublimeCodeIntel#481]
Just spotted this error

```
Traceback (most recent call last):
  File "/opt/sublime_text/sublime_plugin.py", line 442, in is_enabled_
    ret = self.is_enabled()
  File "/home/michael-heuberger/.config/sublime-text-3/Packages/SublimeCodeIntel/SublimeCodeIntel.py", line 1820, in is_enabled
    return super(SublimecodeintelEnableLiveCommand, self).is_enabled(False, False)
  File "/home/michael-heuberger/.config/sublime-text-3/Packages/SublimeCodeIntel/SublimeCodeIntel.py", line 1803, in is_enabled
    enabled = super(SublimecodeintelLiveCommand, self).is_enabled(True)
  File "/home/michael-heuberger/.config/sublime-text-3/Packages/SublimeCodeIntel/SublimeCodeIntel.py", line 1773, in is_enabled
    enabled = enabled and codeintel_enabled(view, True) == active
TypeError: codeintel_enabled() takes from 0 to 1 positional arguments but 2 were given
Traceback (most recent call last):
  File "/opt/sublime_text/sublime_plugin.py", line 438, in is_enabled_
    ret = self.is_enabled(**args)
TypeError: is_enabled() got an unexpected keyword argument 'action'
```

Any suggestions?

**Comment by SOSkr** [comment]
Same error. Sublime Text Build 3065

**Comment by petermac-** [comment]
I'm also seeing this error.

**Comment by TimothyGu** [comment]
Might already be fixed in 30d1b576efec92f737918629bdf94070f8dac0d1

**Comment by rahmatawaludin** [comment]
Me too I'm using 2.1.7+st3 and sublime 3073.

its make my sublime really slow..

**Comment by spectacles** [comment]
You can download CodeComplice via package control . It's a drop in replacement which has this and a lot of other bugs fixed for ST2/3!

**Comment by binarykitchen** [comment]
ah, thanks @spectacles

**Comment by Kronuz** [comment]
Fixed in v2.1.9
